### ISSUE TITLE
search broken after install the latest redmine_tagging plugin

### ISSUE THREAD

**Issue by markshao** [Restream/redmine_tagging#70]
redmine version: 3.2
redmine_tagging version: latest

ActiveRecord::StatementInvalid (Mysql2::Error: Unknown column 'projects.status' in 'where clause': SELECT DISTINCT `wiki_pages`.`created_on`, `wiki_pages`.`id` FROM `wiki_pages` LEFT OUTER JOIN `wiki_page_tags` ON `wiki_page_tags`.`wiki_page_id` = `wiki_pages`.`id` WHERE (projects.status <> 9 AND projects.id IN (SELECT em.project_id FROM enabled_modules em WHERE em.name='wiki')) AND (wikis.project_id IN (7)) AND (((title LIKE '%fadf%') OR (wiki_contents.text LIKE '%fadf%') OR (wiki_page_tags.tag LIKE '%fadf%')))  ORDER BY `wiki_pages`.`created_on` DESC, `wiki_pages`.`id` DESC):
  lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb:171:in `fetch_ranks_and_ids'
  lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb:91:in`search_result_ranks_and_ids'
  lib/redmine/search.rb:127:in `block in load_result_ids'
  lib/redmine/search.rb:125:in`each'
  lib/redmine/search.rb:125:in `load_result_ids'
  lib/redmine/search.rb:115:in`block in load_result_ids_from_cache'
  lib/redmine/search.rb:114:in `load_result_ids_from_cache'
  lib/redmine/search.rb:99:in`result_ids'
  lib/redmine/search.rb:70:in `result_count'
  app/controllers/search_controller.rb:65:in`index'
  lib/redmine/sudo_mode.rb:63:in `sudo_mode'

**Comment by urimaro** [comment]
I've got same error.

### System configuration
**OS**: Ubuntu 14.04.1 LTS (GNU/Linux 3.13.0-36-generic x86_64)
**Ruby**: 2.0.0p645
**Redmine**: 3.1.1
**redmine_tagging**: latest

**Comment by mmjurov** [comment]
Having the same issue, but only if searching by wiki.

```
Started GET "/search?utf8=%E2%9C%93&q=123&scope=all&all_words=&titles_only=&wiki_pages=1&attachments=0&options=0&commit=%D0%9F%D1%80%D0%B8%D0%BD%D1%8F%D1%82%D1%8C" for 212.35.162.69 at 2016-06-08 22:43:37 +0300
Processing by SearchController#index as HTML
  Parameters: {"utf8"=>"✓", "q"=>"123", "scope"=>"all", "all_words"=>"", "titles_only"=>"", "wiki_pages"=>"1", "attachments"=>"0", "options"=>"0", "commit"=>"Принять"}
  Current user: m.zhurov (id=277)
Completed 500 Internal Server Error in 8ms (ActiveRecord: 2.1ms)

ActiveRecord::StatementInvalid (Mysql2::Error: Unknown column 'projects.status' in 'where clause': SELECT DISTINCT `wiki_pages`.`created_on`, `wiki_pages`.`id` FROM `wiki_pages` LEFT OUTER JOIN `wiki_page_tags` ON `wiki_page_tags`.`wiki_page_id` = `wiki_pages`.`id` WHERE (projects.status <> 9 AND projects.id IN (SELECT em.project_id FROM enabled_modules em WHERE em.name='wiki')) AND (((title LIKE '%123%') OR (wiki_contents.text LIKE '%123%') OR (wiki_page_tags.tag LIKE '%123%')))  ORDER BY `wiki_pages`.`created_on` DESC, `wiki_pages`.`id` DESC):
  lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb:171:in `fetch_ranks_and_ids'
  lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb:91:in `search_result_ranks_and_ids'
  lib/redmine/search.rb:127:in `block in load_result_ids'
  lib/redmine/search.rb:125:in `each'
  lib/redmine/search.rb:125:in `load_result_ids'
  lib/redmine/search.rb:115:in `block in load_result_ids_from_cache'
  lib/redmine/search.rb:114:in `load_result_ids_from_cache'
  lib/redmine/search.rb:99:in `result_ids'
  lib/redmine/search.rb:70:in `result_count'
  app/controllers/search_controller.rb:75:in `index'
  lib/redmine/sudo_mode.rb:63:in `sudo_mode'
```

**Comment by thegcat** [comment]
This should be fixed by #72.
